closed/Intel/code/dlrm-99.9/pytorch-cpu/setup_env_offline_ICX.sh

```diff
@@ -0,0 +1,12 @@
+number_threads=`nproc --all`
```

Comment:
could we remove ICX keyword since it also works on SPR?

Reply:
Change key word "ICX" to "AWS", because there is another "setup_env_offline.sh" script in this dir.